# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/mozilla/meta/webgpu/cts/webgpu/shader/execution/expression/call/builtin/textureSampleBias/cts.https.html.ini
# Commit: 3f167f8b3137
# Full Hash: 3f167f8b313749f50f135e189e5854bef7ac0c73
# Author: Erich Gubler <erichdongubler@gmail.com>
# Date: 2025-10-03 18:08 +0000
# Regressor Bug: 1991551
# File Overlap Count: 23
# Overlapping Files: third_party/rust/wgpu-hal/Cargo.toml, supply-chain/audits.toml, third_party/rust/naga/src/back/spv/writer.rs, third_party/rust/naga/src/back/msl/writer.rs, third_party/rust/wgpu-types/src/lib.rs
# Description:
#   Bug 1991551 - build(webgpu): update WGPU to 8c4aebc0c1b75e3f7412dfc180745a0717fc30ec r=webgpu-reviewers,supply-chain-reviewers,nical
# ==============================================================================

diff -r 823439f307a0 -r 3f167f8b3137 testing/web-platform/mozilla/meta/webgpu/cts/webgpu/shader/execution/expression/call/builtin/textureSampleBias/cts.https.html.ini
--- a/testing/web-platform/mozilla/meta/webgpu/cts/webgpu/shader/execution/expression/call/builtin/textureSampleBias/cts.https.html.ini	Fri Oct 03 18:01:30 2025 +0000
+++ b/testing/web-platform/mozilla/meta/webgpu/cts/webgpu/shader/execution/expression/call/builtin/textureSampleBias/cts.https.html.ini	Fri Oct 03 18:08:29 2025 +0000
@@ -10698,7 +10698,7 @@
   [:format="bc4-r-unorm";filt="nearest";mode="c"]
     expected:
       if os == "win" and debug: [PASS, TIMEOUT, NOTRUN]
-      if os == "linux" and debug: [FAIL, TIMEOUT, NOTRUN]
+      if os == "linux" and debug: [PASS, TIMEOUT, NOTRUN]
       if os == "mac" and debug: [FAIL, TIMEOUT, NOTRUN]
 
   [:format="bc4-r-unorm";filt="nearest";mode="m"]
@@ -10710,7 +10710,7 @@
   [:format="bc4-r-unorm";filt="nearest";mode="r"]
     expected:
       if os == "win" and debug: [PASS, TIMEOUT, NOTRUN]
-      if os == "linux" and debug: [FAIL, TIMEOUT, NOTRUN]
+      if os == "linux" and debug: [PASS, TIMEOUT, NOTRUN]
       if os == "mac" and debug: [FAIL, TIMEOUT, NOTRUN]
 
   [:format="bc5-rg-snorm";filt="linear";mode="c"]